### DISCUSSION TITLE
Multi Term Aggregation not supported

### DISCUSSION THREAD

**Discussion by akshaykatyal** [ruflin/Elastica#2076] [Q&A]
I wanted to use multi term aggregation, so is that not supported as the elastic.co supports multi term aggregation? Is there any way to use the multi-term aggregation. 

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-multi-terms-aggregation.html

**Comment by ruflin** [comment]
As you state, it is not supported directly through a class. Likely it can be added pretty easily with an additional class inheriting from AbstractTermsAggregation: https://github.com/ruflin/Elastica/blob/master/src/Aggregation/AbstractTermsAggregation.php (interested to contribute? ;-)).

One thing that is always possible is to send queries / aggs as raw JSON docs: https://elastica.io/example/raw-json-query.html or arrays https://elastica.io/example/raw-array-query.html 

One more method is to use directly the underlying elasticsearch-php client: https://github.com/ruflin/Elastica/blob/master/src/Client.php#L542 / https://github.com/elastic/elasticsearch-php to send queries.
